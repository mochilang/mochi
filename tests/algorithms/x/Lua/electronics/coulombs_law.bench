{
  "duration_us": 281,
  "memory_bytes": 848,
  "name": "main"
}