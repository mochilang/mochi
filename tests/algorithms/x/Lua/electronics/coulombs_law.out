{"__name": GenType1}
{"force": 33705.0}
{"__order": table: 0x5557183845d0}
{"__name": GenType4}
{"distance": 116112.01820814}
{"__order": table: 0x555718384e90}
{"__name": GenType2}
{"__order": table: 0x555718380eb0}
{"charge1": 0.0008900756564308}